tests/unit/parsers/test.manifestjson.js

```diff
@@ -272,21 +272,14 @@ describe('ManifestJSONParser', () => {
           }),
         ])
       );
-      expect(manifestJSONParser.collector.errors).toEqual(
-        expect.arrayContaining([
-          expect.objectContaining({
-            code: 'MANIFEST_BAD_PERMISSION',
-            dataPath: '/permissions',
-          }),
-        ])
-      );
+      expect(manifestJSONParser.collector.errors).toEqual([]);
     });
 
     it('ignores host_permissions manifest key in MV2', () => {
       const addonLinter = new Linter({ _: ['bar'] });
       const json = validManifestJSON({
         manifest_version: 2,
-        host_permissions: ['*://example.org/*'],
+        host_permissions: ['*://example.org/*', 'foo'],
```

Comment:
Do you really need to change this test case here?

Reply:
the additional 'foo' here is to be sure that we are actually ignoring the host_permissions and not reporting any validation error ('foo' is invalid as an host permission, the other one is not), and so technically we may actually remove the other one (`'*://example.org/*'`) and just keep the invalid one (`'foo'`).